### PR TITLE
Lead and lead acetate

### PR DESCRIPTION
**Pull request by CursedBirb**
## About The Pull Request

Adds recipes for lead and lead acetate

## Why It's Good For The Game

Lead itself is really hard to get. Only way is to find either lead pipe or lead cell in maint and as maint loot it can not spawn. Also pipe can be grinded for 20u and cell for 15u so it is not a lot. I made recipes so bizzare that it will not be created often. Lead Acetate recipe was requested by one person. It is not a good poison so i do not think it would be a problem for it to be produced normally. For your own health do not look at lead recipe EOBGames
## Changelog

:cl: Cursed Birb
expansion: Adds recipes for lead and lead acetate

/:cl:

### CODE REVIEW

**Comment by Amrabol** [comment]
Looks like i need to spend some time fixing issues to go above zero

**Comment by MrMelbert** [comment]
IIRC Lead was intentionally difficult to come across because it made some of the new drugs.

**Comment by Amrabol** [comment]
> IIRC Lead was intentionally difficult to come across because it made some of the new drugs. Not gonna block the pr or anything, just of note.

I made it annoying to prevent people from making too much of it as that might lead to entire rounds of someone invisible. Previous limited amount of lead limited that but now as there is recipe i had to give them indirect suggestion that maybe they should not try to make a lot

**Comment by D4C-420** [comment]
> made it annoying to prevent people from making too much of it as that might lead to entire rounds of someone invisible.

Even bizarre recipes can be speed run with enough practice.

**Comment by Amrabol** [comment]
> > made it annoying to prevent people from making too much of it as that might lead to entire rounds of someone invisible.
> 
> Even bizarre recipes can be speed run with enough practice.

I know but only other way would have to make it need Wittel or Hyper-Plasmium Oxide

**Comment by Time-Green** [comment]
Lead is a gated resource, and it’s recipy’s were designed with that in mind. More common for the sake of more common is bad design 

Also the recipy is literal alchemy

@Krysonism thoughts?

File: code/modules/reagents/chemistry/recipes/toxins.dm [hunk 1]
```diff
@@ -405,3 +405,23 @@
 	purity_min = 0.4
 	reaction_flags = REACTION_PH_VOL_CONSTANT
 	reaction_tags = REACTION_TAG_EASY | REACTION_TAG_DAMAGING | REACTION_TAG_OTHER
+
+/datum/chemical_reaction/leadacetate
+	results = list(/datum/reagent/toxin/leadacetate = 1)
+	required_reagents = list(/datum/reagent/consumable/ethanol = 1, /datum/reagent/lead = 1, /datum/reagent/hydrogen_peroxide = 1)
+	mix_message = "Just looking at beaker makes you ear and head hurt."
```
Comment (ZephyrTFA) [inline code/modules/reagents/chemistry/recipes/toxins.dm:412]:
```suggestion
	mix_message = "Just looking at beaker makes your ear and head hurt."
```

**Comment by Kylerace** [comment]
gbp too low for features sorry

**Pull request closed without merging**